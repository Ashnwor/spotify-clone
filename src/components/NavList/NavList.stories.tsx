import React from 'react';
import PlaylistList from './NavList';

export default {
  title: 'Playlist List',
};

export const playlistList = () => <PlaylistList />;
