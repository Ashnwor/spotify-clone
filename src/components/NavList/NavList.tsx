import styles from './NavList.module.scss';
import NavListItem from '../NavListItem/NavListItem';

const NavList: React.FC = () => {
  return (
    <div className={styles['playlist-list']}>
      <NavListItem text="playlist placeholder" />
      <NavListItem text="playlist placeholder" />
      <NavListItem text="playlist placeholder" />
    </div>
  );
};

export default NavList;
