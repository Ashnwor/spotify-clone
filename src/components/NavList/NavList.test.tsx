import React from 'react';
import { shallow } from 'enzyme';
import NavList from './NavList';

it('runs correctly', () => {
  const wrapper = shallow(<NavList />);
  expect(wrapper).toMatchSnapshot();
});
