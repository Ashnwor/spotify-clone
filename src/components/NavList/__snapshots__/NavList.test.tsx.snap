// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="playlist-list"
>
  <NavListItem
    text="playlist placeholder"
  />
  <NavListItem
    text="playlist placeholder"
  />
  <NavListItem
    text="playlist placeholder"
  />
</div>
`;
